### PR TITLE
LVNOD-109_set_eslint_on_Travis_on_front_end

### PR DESCRIPTION
**Pull request by yarvanya**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-109)

## Code reviewers

- [ ] @denisbodnar
- [ ] @AlbaMontis
- [ ] @shyyka

### Second Level Review

- [ ] @lhalam
- [x] @4asof4ik

## Summary of issue

I've added eslint rules for Travis CI.

## CHECK LIST
- [x]  СI passed
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 before_script:
   - npm install -g npm@'>=4.0.0'
   - npm install -g sass-lint
+  - npm install -g eslint
+  - npm install -g babel-eslint
+  - npm install -g eslint-plugin-react@latest
 
 script:
   - sass-lint -v -q 'app/**/*.scss'
+  - eslint ./app/src
```
Comment (4asof4ik) [inline .travis.yml]: Need add for test folder too.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            "arrowFunctions": true,
+            "jsx": true
+        },
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "parser": "babel-eslint",
+    "rules": {
```
Comment (4asof4ik) [inline .eslintrc.json:21]: Empty line  in the end of file

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 2 above)
Comment (yarvanya) [reply .eslintrc.json:21]: I have this rule.

File: .travis.yml (hunk 1 above)
Comment (yarvanya) [reply .travis.yml]: Done.

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        "indent": [
+            "error", 2, { "SwitchCase": 1 }
```
Comment (AlbaMontis) [inline .eslintrc.json:23]: Did you check indentation in this file? :)

Comment (yarvanya) [reply .eslintrc.json:23]: Yes, I did))

Comment (AlbaMontis) [reply .eslintrc.json:23]: So why do you use 4 spaces here? Is this file special?

**Review by 4asof4ik — APPROVED** [review]

**Pull request merged**